packages/@uppy/dashboard/src/components/VirtualList.js

```diff
@@ -70,7 +70,6 @@ class VirtualList extends Component {
     window.addEventListener('resize', this.handleResize)
   }
 
-  // TODO: can we use a different method here?
```

Comment:
maybe we should either keep the TODO or remove the eslint disable, because this method is really deprecated and should be removed, so it's relevant still?